### PR TITLE
Updated release strategies

### PR DESCRIPTION
**Pull request by Sgtpluck**

### CODE REVIEW

File: _pages/release-strategies.md [hunk 1]
```diff
@@ -6,106 +6,114 @@ sticky_sidenav: true
...
-## Small, Iterative Releases
+## Small, iterative releases
 
 We recommend on all projects to get into the habit of releasing software as early and often as possible. The first
-iteration of your software should not aim to have full functionality; a better goal for the very first release is to have as little
-functionality as possible.
+iteration of your software should not aim to have full functionality; a better goal for the very first release is to have as [little
+functionality as possible](https://18f.gsa.gov/2017/01/11/the-best-way-to-build-big-is-to-start-small/#start-small-make-it-end-to-end). 
+A small-and-quick first release can help you [test your deployment pipeline and environments, while setting expectations for a rapid,
+iterative release cadence](https://blog.thepete.net/blog/2019/10/04/hello-production/).
```
Comment (Sgtpluck) [inline _pages/release-strategies.md:15]: i think we need to build out the recommendation FOR small iterative releases, but that was outside the scope of this original write-up.

Comment (tbaxter-18f) [reply _pages/release-strategies.md:15]: I think we've written so much on it before a link (or links) is fine.

File: _pages/release-strategies.md [hunk 2]
```diff
@@ -6,106 +6,114 @@ sticky_sidenav: true
...
+#### The partner doesn't want to pick a cohort of controlled users
 
 Understanding why a partner doesn't want to single out some controlled users is important. They may be nervous about creating inequity (or
-a perception of inequity) that might have some polical blowback. Or perhaps they feel overwhelmed about the effort required to narrow down
+a perception of inequity) that might have some political blowback. Perhaps they feel overwhelmed about the effort required to narrow down
 to a subset of users, and so feel it's simpler to release to all users at once.
 
 *Mitigation strategies:*
-* Work with your partner to fully understand their fears. 
+* Listen to and discuss fears with your partner to fully understand their reluctance.
```
Comment (tbaxter-18f) [inline _pages/release-strategies.md:100]: nice

File: _pages/release-strategies.md [hunk 3]
```diff
@@ -6,106 +6,114 @@ sticky_sidenav: true
...
 
-#### A Long, Arduous ATO Process
+#### A long, arduous ATO process
 
 Partners have a valid fear of the ATO process, and may be nervous that they must repeat the ATO process every time a change is made to the
 system. That perspective easily lends itself to fear of iterative releases. 
 
 *Mitigation strategies:*
-* Engage security or ATO personnel early in the development process, or, ideally, embed someone onto the project team who can help advise.
+* Engage security and ATO personnel early in the development process, or, ideally, embed someone onto the project team who can help advise. This strategy (among other improvements) helped GSA bring average ATO time from [six months to thirty days](https://18f.gsa.gov/2018/07/19/taking-the-ato-process-from-6-months-to-30-days/).
```
Comment (tbaxter-18f) [inline _pages/release-strategies.md:110]: nice shoutout here

**Review by tbaxter-18f — APPROVED** [review]
Looks great.

**Pull request merged**